### PR TITLE
[Shaz|Jaicharan]  Ignoring Vulnerablities in Snyk 

### PR DESCRIPTION
**Pull request by shaz-ahammed**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,10 @@
 # Created by https://www.toptal.com/developers/gitignore/api/visualstudio,reactnative,react,git,ruby,yarn,node,xcode,android,intellij,fastlane
 # Edit at https://www.toptal.com/developers/gitignore?templates=visualstudio,reactnative,react,git,ruby,yarn,node,xcode,android,intellij,fastlane
 
+# OSX
```
Comment (vraravam) [inline .gitignore]: please re-generate from gitignore.io with the extra section headers

File: .gitignore [hunk 2]
```diff
@@ -12,6 +16,26 @@ local.properties
 # Log/OS Files
 *.log
 
+# Xcode
```
Comment (vraravam) [inline .gitignore]: pls regenerate

**Review by vraravam — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (jai-charan) [reply .gitignore]: Changes made

File: .gitignore [hunk 3]
```diff
@@ -1,6 +1,35 @@
 # Created by https://www.toptal.com/developers/gitignore/api/visualstudio,reactnative,react,git,ruby,yarn,node,xcode,android,intellij,fastlane
```
Comment (vraravam) [inline .gitignore]: I expected this line to change @jai-charan

**Review by vraravam — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 3 above)
Comment (jai-charan) [reply .gitignore]: like mentioning all sections that has been ignored?

Comment (vraravam) [reply .gitignore]: when you regenerate the page, please take the full block from the "starting" line to the "ending" line

Comment (jai-charan) [reply .gitignore]: okay...I will regenerate the file

File: .gitignore [hunk 4]
```diff
@@ -1004,32 +1026,4 @@ FodyWeavers.xsd
...
-coverage.out
-/sonar-project.properties
-
-# http://www.sonarlint.org/commandline/
-# SonarLint working directories, configuration files (including credentials)
-.sonarlint/
-
-#App specific
-/test-reports/jest-html-reporters-attach
-/test-reports/test-report.html
```
Comment (vraravam) [inline .gitignore]: I think these 3 lines need to be left at the end of the generated file?

**Review by vraravam — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 4 above)
Comment (jai-charan) [reply .gitignore]: yeah

**Review by vraravam — APPROVED** [review]

**Pull request merged**